content/authors/tencate/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Balder ten Cate
+lastname: Ten Cate
```

Comment:
if you prefer your last name to be sorted according to "Cate", then that's your chance here. ;-)

Reply:
Hi Christian, thanks for catching this - I fixed it.